tests_vcr/test_trello_data_with_vcr.py

```diff
@@ -0,0 +1,64 @@
+import vcr
```

Comment:
The first two "tests" are just using the `requests` library directly, so seem pointless.

Also, I don't think there's any value in having both VCR yaml files to hold canned responses and also your own json files used by `mock_responses.py`. Pick one solution and stick with it.

Reply:
Removed VCR